modules/nixos/monitoring/alert-rules.nix

```diff
@@ -0,0 +1,191 @@
+{ lib }:
+lib.mapAttrsToList
+  (name: opts: {
+    alert = name;
+    expr = opts.condition;
+    for = opts.time or "2m";
+    labels = { };
+    annotations.description = opts.description;
+    # for matrix alert-receiver
+    annotations.summary = opts.description;
+  })
+  ((lib.genAttrs [
+    "borgbackup-github-org"
+    "borgbackup-nixpkgs-update"
```

Comment:
I just thought about renaming them like this:
borgbackup-job-github-org.service
Because when they fail than you can just copy and paste the name from prometheus to get the systemctl status.